CommandCenter.m

```diff
@@ -73,6 +73,16 @@ function CommandCenter_OpeningFcn(hObject, eventdata, handles, varargin)
     % Means it already exists, so do nothing
     return
 end
+% Check compatibility
+if true || verLessThan('matlab','9.4') % R2018a
```

Comment:
This seems like a situation that could happen with any number of functions for outdated Matlabs. I guess this is a pretty big one that crosses a lot of Command Center, but is there a more general way to handle errors of this type? Another common one I can think of is GUI updates, which newer Matlabs seem to make a lot of.

Reply:
ha im glad you highlighted this line because i forgot to take out my "or true" for testing :P 

What do you mean "general" way? Graphics haven't changed that much recently - they just moved where they positioned some menu stuff. The super problematic changes are when they change underlying language characteristics like how implicit constructor calls are made or big changes in core components like when figures became classes. These changes would be hard to detect at launch, and may not even be obvious at runtime.

That being said, a more general approach would be to have components define their version restrictions independently. But, that would require checks during runtime, which perhaps is fine. We could implement like a "compatible" method call upon loading. However, given how we currently find these issues the current implementation might be more desirable. Randomly find an issue...investigate why it happens...update version threshold for the application or fix issue to be more backwards compatible.

Reply:
I think it's better to update the version threshold (or better yet, fix issues for backwards compatibility as possible) as issues come up as @mwalsh161 suggested.